packages/playground/src/components/manage_gateway_dialog.vue

```diff
@@ -69,9 +68,8 @@
               {{ item.name }}
             </template>
 
-            <template #[`item.tls_passthrough`]="{ item }">
-              {{ item.tls_passthrough ? "Yes" : "No" }}
-            </template>
+            <template #[`item.tls_passthrough`]="{ item }"> {{ item.tls_passthrough ? "Yes" : "No" }} </template>Manage
+            Domains
```

Comment:
not clear what you are doing here; please explain

Reply:
removed it as manage domains was added by accident